lib/smart_proxy_dns_infoblox/plugin_configuration.rb

```diff
@@ -1,12 +1,22 @@
 module Proxy::Dns::Infoblox
   class PluginConfiguration
     def load_classes
+      require 'ipaddr'
+      require 'resolv'
       require 'infoblox'
       require 'dns_common/dns_common'
       require 'smart_proxy_dns_infoblox/dns_infoblox_main'
       require 'smart_proxy_dns_infoblox/infoblox_member_dns'
     end
 
+    def get_dns_server_ip(dns_server)
+      begin
+        IPAddr.new(dns_server).to_s
+      rescue IPAddr::InvalidAddressError
+        Resolv.getaddresses(dns_server).first
```

Comment:
This needs to handle errors (timeout, resolv error) and we should error out with a fatal error nicely explaining what just happened.